src/main/java/com/ait/lienzo/client/core/shape/AbstractDirectionalMultiPointShape.java

```diff
@@ -97,6 +98,11 @@ public T setCorrectionOffset(final double offset)
         return list;
     }
 
+    @Override
+    public Point2D adjustPoint(double x, double y, double deltaX, double deltaY) {
+        return null;
```

Comment:
IMO it is better to return the point even if there is no need for adjusting it, better than return null, WDYT ?